scrapinghub/client/jobs.py

```diff
@@ -22,30 +22,42 @@ class Jobs(object):
     instance to get a :class:`Jobs` instance. See :attr:`Project.jobs` and
     :attr:`Spider.jobs` attributes.
 
-    :ivar projectid: an integer project id.
+    :ivar project_id: a string project id.
     :ivar spider: :class:`Spider` object if defined.
 
     Usage::
 
         >>> project.jobs
-        <scrapinghub.client.Jobs at 0x10477f0b8>
+        <scrapinghub.client.jobs.Jobs at 0x10477f0b8>
         >>> spider = project.spiders.get('spider1')
         >>> spider.jobs
-        <scrapinghub.client.Jobs at 0x104767e80>
+        <scrapinghub.client.jobs.Jobs at 0x104767e80>
     """
 
-    def __init__(self, client, projectid, spider=None):
-        self.projectid = projectid
+    def __init__(self, client, project_id, spider=None):
+        self.project_id = project_id
         self.spider = spider
         self._client = client
-        self._project = client._hsclient.get_project(projectid)
+        self._project = client._hsclient.get_project(project_id)
+
+    def count(self, spider=None, state=None, has_tag=None, lacks_tag=None,
+              startts=None, endts=None, **params):
+        """Count jobs with a given set of filters.
+
+        :param spider: (optional) filter by spider name.
+        :param state: (optional) a job state, a string or a list of strings.
+        :param has_tag: (optional) filter results by existing tag(s), a string
+            or a list of strings.
+        :param lacks_tag: (optional) filter results by missing tag(s), a string
+            or a list of strings.
+        :param startts: (optional) UNIX timestamp at which to begin results,
+            in millisecons.
+        :param endts: (optional) UNIX timestamp at which to end results,
+            in millisecons.
+        :param \*\*params: (optional) other filter params.
 
-    def count(self, **params):
-        """Count jobs for a given set of parameters.
-
-        :param \*\*params: (optional) a set of filters to apply when counting
-            jobs (e.g. spider, state, has_tag, lacks_tag, startts and endts).
         :return: jobs count.
+        :rtype: int.
```

Comment:
I don't think rtype value should have a trailing dot, please check across the repo